src/index.js

```diff
@@ -1,6 +1,25 @@
 import './style.css';
-import { scoreList, loadList } from './load.js';
+import { addScore, displayScores } from './api.js';
 
-const ulContain = document.querySelector('#score_list');
+const form = document.querySelector('#form');
+const refresh = document.getElementById('refresh_btn');
+const yourname = document.querySelector('#yourname');
+const yourscore = document.querySelector('#yourscore');
 
-loadList(scoreList, ulContain);
+refresh.addEventListener('click', displayScores);
+
+form.addEventListener('submit', (e) => {
+  const name = yourname.value;
+  const score = yourscore.value;
+  if (name !== '' && score !== '') {
+    const data = {
+      user: name,
+      score,
+    };
+    addScore(data);
+    yourname.value = '';
+    yourscore.value = '';
+  }
+  e.preventDefault();
+  displayScores();
+});
```

Comment:
- [ ] Hi, this event listener won't work because in your index.html file the `input` element for submitting has a `type="button"` which is not correct if you want to set up a `"submit"` event Listeners. please change `input` element to have a `type="submit"`.  This is why adding a score doesn't currently work this event listener never fires. please kindly make that change 🙏